clients/admin-ui/src/features/dataset/DatasetTable.tsx

```diff
@@ -68,7 +68,7 @@ const DatasetsTable = () => {
               <Td pl={1}>{dataset.name}</Td>
               <Td pl={1}>{dataset.fides_key}</Td>
               <Td pl={1}>{dataset.description}</Td>
-              {features.plus ? (
+              {features.plus && classifyDataset?.status ? (
```

Comment:
Cool, thanks @allisonking ! Will get this addressed, I noticed something was off because the cursor flipped as well on hover. I think I originally had something in `ClassificationStatusBadge` but got some complaints there about an unnecessary return and then another complaint for not returning null so just moved it higher 😅

Reply:
Moved inside the TD and validated the table is looking sharper now 
<img width="1129" alt="image" src="https://user-images.githubusercontent.com/6758661/230239303-cd2a8046-8e27-4c66-bcd1-70d1d9af7ba6.png">
Thanks for catching that and the suggestions!

Reply:
ah yeah I think it'd be better to have `features.plus` render the `<Td>` always, but have `classifyDataset?.status` decide whether or not the contents of the `<Td>` render. this is mostly because if you look at the `<Th>` above, it only renders "Status" if plus is around, so we want to keep the `Th` and `Td`s consistent so the table doesn't get confused about how many columns it has

this becomes noticeable (but barely!) in non-plus, where you've got one less `Th` than `Td` and so you get this lil gap way on the right of the table
![image](https://user-images.githubusercontent.com/24641006/230455583-4cf13a55-bb22-461d-9622-ce809d4edab4.png)

Reply:
yep, makes sense to me too!

Okay, so now here is `fidesplus`:
<img width="295" alt="image" src="https://user-images.githubusercontent.com/6758661/230506989-df0bb9ce-de2b-4edb-96b6-6fdbf8cf23e6.png">


and here is `fides`:
<img width="1098" alt="image" src="https://user-images.githubusercontent.com/6758661/230507477-383f78aa-f9b1-434b-8ee2-867504771e7d.png">

It still has that slight gap but there the `Th` quantity matches the `Td` so I think it's just a spacing thing at this point. Thanks again for your help!